production/grafana-agent-mixin/dashboards.libsonnet

```diff
@@ -112,32 +112,13 @@ local template = grafana.template;
             (
               rate(prometheus_remote_storage_highest_timestamp_in_seconds{cluster=~"$cluster", namespace=~"$namespace", container=~"$container"}[5m])
               -
-              ignoring(url, instance_group_name, remote_name) group_right(pod)
+              ignoring(url, remote_name) group_right(pod)
               rate(prometheus_remote_storage_queue_highest_sent_timestamp_seconds{cluster=~"$cluster", namespace=~"$namespace", container=~"$container"}[5m])
             )
           |||,
           legendFormat='{{cluster}}:{{pod}}-{{instance_group_name}}-{{url}}',
         ));
 
-      local samplesRate =
-        graphPanel.new(
-          'Rate, in vs. succeeded or dropped [5m]',
-          datasource='$datasource',
-          span=12,
-        )
-        .addTarget(prometheus.target(
-          |||
-            rate(
-              prometheus_remote_storage_samples_in_total{cluster=~"$cluster", namespace=~"$namespace", container=~"$container"}[5m])
-            -
-              ignoring(url, instance_group_name, remote_name) group_right(pod)
-              rate(prometheus_remote_storage_succeeded_samples_total{cluster=~"$cluster", namespace=~"$namespace", container=~"$container"}[5m])
-            -
-              rate(prometheus_remote_storage_dropped_samples_total{cluster=~"$cluster", namespace=~"$namespace", container=~"$container"}[5m])
-          |||,
-          legendFormat='{{cluster}}:{{pod}}-{{instance_group_name}}-{{url}}',
-        ));
-
```

Comment:
I removed this one because the succeeded metric was removed(https://github.com/prometheus/prometheus/blob/50d506ef489c31feebf940c4962e44c87f6c835e/CHANGELOG.md#2230--2020-11-26) and I didn't see a great replacement.